core/block_dragger.js

```diff
@@ -227,3 +231,29 @@ Blockly.BlockDragger.prototype.updateCursorDuringBlockDrag_ = function() {
     }
   }
 };
+
+/**
+ * TODO (fenichel): Consider putting this in utils.
+ * Convert a coordinate object from pixels to workspace units, including a
```

Comment:
I've forgotten - for "pixel coordinates" is 0,0 still the top corner of the workspace?